lxd/instance/drivers/driver_qemu.go

```diff
@@ -4931,7 +4933,7 @@ func (d *qemu) devlxdEventSend(eventType string, eventMessage interface{}) error
 
 	agent, err := lxdClient.ConnectLXDHTTP(nil, client)
 	if err != nil {
-		logger.Errorf("Failed to connect to lxd-agent on %s: %v", d.Name(), err)
+		d.common.logger.Error(fmt.Sprintf("Failed to connect to lxd-agent on %s: %v", d.Name(), err))
```

Comment:
@kevtheappdev in this case, to add contextual logging we would do:

```
d.logger.Error("Failed to connect to lxd-agent", log.Ctx{"err": err})
```

This uses the logger's main context and adds a one-time additional context for value for the error.